devtools/azure/ci.yml

```diff
@@ -1,23 +1,37 @@
 steps:
-- script: |
-    yarn
-  displayName: 'Install npm dependencies'
-- script: |
-    yarn workspace @ckb-lumos/helpers build
-    yarn workspace @ckb-lumos/rpc build
-    yarn workspace @ckb-lumos/common-scripts build
-    yarn workspace @ckb-lumos/hd build
-    yarn workspace @ckb-lumos/hd-cache build
-    yarn workspace @ckb-lumos/base test
-    yarn workspace @ckb-lumos/common-scripts test
-    yarn workspace @ckb-lumos/config-manager test
-    yarn workspace @ckb-lumos/hd test
-    yarn workspace @ckb-lumos/hd-cache test
-    yarn workspace @ckb-lumos/helpers test
-    yarn workspace @ckb-lumos/indexer test
-    yarn workspace @ckb-lumos/transaction-manager test
-    yarn workspace @ckb-lumos/rpc test
-    yarn workspaces run fmt
-    yarn workspaces run lint
-    git diff --exit-code
-  displayName: 'Generic tests'
+  - script: |
+      yarn
+    displayName: "Install npm dependencies"
+  - script: |
+      nohup yarn workspace @ckb-lumos/testkit start &
+      nohup ./ckb-indexer -c http://127.0.0.1:8118/rpc -l 127.0.0.1:8120 -s indexer-store-tmp &
+      echo '{
+          "id": 2,
+          "jsonrpc": "2.0",
+          "method": "get_tip"
+      }' \
+      | tr -d '\n' \
+      | curl -H 'content-type: application/json' -d @- \
+      http://localhost:8120
```

Comment:
the non-hangup mode for both commands may cause the ckb-indexer to start before the CKB node